core/src/main/java/com/linecorp/armeria/common/util/Functions.java

```diff
@@ -194,5 +196,25 @@
         };
     }
 
+    /**
+     * Converts the specified {@link Supplier} into a {@link Callable}
+     */
+    public static <T> Callable<T> getToCall(Supplier<T> supplier) {
+        return supplier::get;
+    }
+
+    /**
+     * Converts the specified {@link Callable} into a {@link Supplier}
+     */
+    public static <T> Supplier<T> callToGet(Callable<T> callable) {
+        return () -> {
+            try {
+                return callable.call();
+            } catch(Exception e) {
```

Comment:
Could you use Throwables.throwIfUnchecked() first so that unchecked exceptions are thrown as is?

Reply:
Yes, I can.
`Throwables.throwIfUnchecked ()` handles RuntimeException and Error but not all exceptions. But, this solution needs to manage post logic.
If `callable.call()` method throws an exception, I think that method should throw an exception in catch block. What do you think?

```java
} catch (Throwable e) {
      Throwables.throwIfUnchecked(e);
      throw new RuntimeException("fromCallable call exception", e);
}
```